### DISCUSSION TITLE
Cleanup when formula is uninstalled

### DISCUSSION THREAD

**Discussion by tillkruss** [Homebrew/discussions#1064] [Everyday usage] [answered]
Hi there!

is there a way for a formula to remove some of its files (outside its cellar) when it’s uninstalled?

I’ve searched for any references, but can’t find anything.

**Comment by SMillerDev** [comment, marked as answer]
There is not. Only configuration and user data should be outside the Cellar and people don't like it if you remove those things.

**Reply by tillkruss** [reply]
Thanks, that's good to know.

Is there any way to call `brew uninstall <formula>` so that the the `etc/` files are deleted as well?

```
brew uninstall foobar
Uninstalling /opt/homebrew/Cellar/foobar/HEAD... (3 files, 929.3KB)

Warning: The following relay configuration files have not been removed!
If desired, remove them manually with `rm -rf`:
  /opt/homebrew/etc/foobar
  /opt/homebrew/etc/foobar/foobar.ini
```

**Reply by SMillerDev** [reply]
No, if that is desired you can remove them manually with `rm -rf` 😉

**Reply by tillkruss** [reply]
Thanks @SMillerDev!
